src/main/java/seedu/duke/command/DatabaseStorage.java

```diff
@@ -0,0 +1,206 @@
+package seedu.duke.command;
+
+import java.io.IOException;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import seedu.duke.exceptions.InvalidDatabaseLineException;
+import seedu.duke.exceptions.InvalidModuleException;
+import seedu.duke.exceptions.InvalidModuleMappingException;
+import seedu.duke.exceptions.InvalidUniversityException;
+import seedu.duke.module.Module;
+import seedu.duke.module.ModuleMapping;
+import seedu.duke.university.University;
+import seedu.duke.parser.DatabaseParser;
+import seedu.duke.ui.Ui;
+
+public class DatabaseStorage {
+    private static Logger logger = Logger.getLogger("DatabaseStorage");
+    private static final String DATABASE_FILE_PATH = "./data/data.csv";
+
+    private static final String FILE_DIRECTORY = "data";
+
+    private static final String FILE_PATH = "data/data.csv";
+
+    private static final String PARTNER_UNVIERSITY_COUNTRY = "nil";
+
+
+    /**
+     * Helps user set up the database.
+     */
+    public static void createDatabase() throws IOException {
+        File fileDir = new File(FILE_DIRECTORY);
+        if (!fileDir.exists()) {
+            fileDir.mkdir();
+        }
+        File file = new File(FILE_PATH);
+        if (!file.exists()) {
+            file.createNewFile();
+            Path target = Path.of(FILE_PATH);
+            URL source = new URL("https://raw.githubusercontent.com/AY2223S1-CS2113-W13-2/tp/master/data/data.csv");
+            try (InputStream in = source.openStream()) {
+                Files.copy(in, target, StandardCopyOption.REPLACE_EXISTING);
+            }
+        }
+    }
+
+    /**
+     * Loads data from data.csv into database.
+     */
```

Comment:
Good habit for adding comments here and there. But in this case, it's unnecessary if the method name is already self-describing.

In general we only want to add comments when necessary. There are various reasons, but the main one is to prevent outdated comments. If there are too many comments it could become hard to maintain, so keep in mind.